### PR TITLE
Add ProvisioningTicketUrl

### PR DESCRIPTION
**Pull request by zzanol**
Added ProvisioningTicketUrl to Connection class.  ProvisioningTicketUrl (provisioning_ticket_url) is returned when getting connections of with specific strategies, including 'google-apps'.

### CODE REVIEW

File: src/Auth0.ManagementApi/Models/Connection.cs [hunk 1]
```diff
@@ -30,5 +30,11 @@ public class Connection : ConnectionBase
         /// </summary>
         [JsonProperty("strategy")]
         public string Strategy { get; set; }
+
+        /// <summary>
+        /// The provisioning ticket URL
```
Comment (frederikprijck) [inline src/Auth0.ManagementApi/Models/Connection.cs:35]:
```suggestion
        /// The provisioning ticket URL for AD / LDAP connections
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Comment by frederikprijck** [comment]
Thanks for the PR, added a small comment but happy to merge once resolved.

File: src/Auth0.ManagementApi/Models/Connection.cs (hunk 1 above)
Comment (zzanol) [reply src/Auth0.ManagementApi/Models/Connection.cs]: Done, thank you

**Review by frederikprijck — APPROVED** [review]

**Comment by zzanol** [comment]
Anything else needed to get this merged?

**Comment by frederikprijck** [comment]
No, sorry about that. Thanks for the PR. Will see to cut a release soon, might need to see if we need to incorporate the other open PR's first.

**Pull request merged**